src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getArea() {
+        return radius * radius * Math.PI;
+    }
+
+    public String draw() {
+        return "Figure: Circle, Area: "
+                + getArea()
+                + " units, Radius: "
+                + radius
+                + " units, "
+                + " Figure Color "
+                + getColor();
```

Comment:
The 'draw()' method should be part of a separate interface as it is conceptually different from 'getArea()'. Consider creating an interface like 'Drawable' with a 'draw()' method and implement it in the 'Circle' class.